notebooks/03_raster_zonal_stats.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        aggregation=dict(func=["mean", "max", "min", "std"], column="elevation"),
</pre><p> </p><p>Just sanity checking. Is the <code>elevation</code> column name from the raster itself (i.e. the raster has multiple bands and each band is named)?</p><p> </p><p>Double checking as well that our implementation takes into account the possibility of multiple bands in a raster right? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/geowrangler/pull/104/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Actually right now it doesn't. The underlying `rasterstats.zonal_stats` [method](https://pythonhosted.org/rasterstats/manual.html#zonal-statistics) only handles 1 band at a time. 

We could create an enhancement so that it can loop through multiple bands -- we'll have to modify the aggregation spec to specify a 'band' instead of a 'column' name. (right now, the band number is specified as a part of the 'extra_args' parameter -- with a default of 1.